ansible/roles/nginx/templates/rogeros_auth_proxy.conf.j2

```diff
@@ -9,6 +7,192 @@ upstream chronos {
 }
 
 server {
+       listen 80;
+       server_name marathon.* chronos.* marathons.* chronoss.*;
+       rewrite ^ https://$host$request_uri permanent;
+}
+
+server {
+        listen 443 ssl;
+        server_name marathon.* marathons.*;
+
+        ssl on;
+        #access_log      /var/log/upstart/nginx.log;
+        #error_log       /var/log/upstart/nginx.log;
+
+        ssl_certificate         {{ nginx_certs_dir }}/nginx.in.crt;
+        ssl_certificate_key     {{ nginx_certs_dir }}/nginx.in.key;
+        ssl_protocols           TLSv1 TLSv1.1 TLSv1.2;
+        ssl_ciphers             HIGH:!aNULL:!MD5;
+
+
+        #proxy_ssl_trusted_certificate /etc/ssl/certs/trusted_ca_cert.crt;
+
+        #proxy_ssl_verify        on;
```

Comment:
We could ignore these commented directives for now as they depend on the "proxy_ssl_trusted_certificate" to be used.